templates/jobs.yaml

```
@@ -577,6 +577,7 @@ spec:
           {{- toYaml $.Values.st2.packs.volumes.configs | nindent 10 }}
         {{- end }}
         # st2canary job does not support extra_volumes. Let us know if you need this.
+      restartPolicy: OnFailure
```

```suggestion
      restartPolicy: Never
```

If the test pod fails, it should not restart -- it should just stay in the failed state.